src/main/java/CarRacing/domain/CarName.java

```diff
@@ -0,0 +1,30 @@
+package CarRacing.domain;
+
+public class CarName {
+    private static final int LIMIT_LENGTH_OF_CAR_NAME = 5;
+
+    private String name;
+
+    public CarName(String name) {
+        validateName(name);
+        setName(name);
+    }
+
+    public String Name() {
```

Comment:
```suggestion
    public String name() {
```
일관성있게 이 부분도 카멜케이스로 작성 부탁드립니다